cuda/ubi9-python-3.11/Dockerfile

```diff
@@ -12,26 +12,26 @@ LABEL name="odh-notebook-cuda-ubi9-python-3.11" \
       io.openshift.build.image="quay.io/opendatahub/workbench-images:cuda-ubi9-python-3.11"
 
 # Install CUDA base from:
-# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/12.1.1/ubi9/base/Dockerfile
+# https://gitlab.com/nvidia/container-images/cuda/-/tree/master/dist/12.4.1/ubi9/base
 USER 0
 WORKDIR /opt/app-root/bin
 
-ENV NVARCH x86_64
-ENV NVIDIA_REQUIRE_CUDA "cuda>=12.1 brand=tesla,driver>=470,driver<471 brand=unknown,driver>=470,driver<471 brand=nvidia,driver>=470,driver<471 brand=nvidiartx,driver>=470,driver<471 brand=geforce,driver>=470,driver<471 brand=geforcertx,driver>=470,driver<471 brand=quadro,driver>=470,driver<471 brand=quadrortx,driver>=470,driver<471 brand=titan,driver>=470,driver<471 brand=titanrtx,driver>=470,driver<471 brand=tesla,driver>=525,driver<526 brand=unknown,driver>=525,driver<526 brand=nvidia,driver>=525,driver<526 brand=nvidiartx,driver>=525,driver<526 brand=geforce,driver>=525,driver<526 brand=geforcertx,driver>=525,driver<526 brand=quadro,driver>=525,driver<526 brand=quadrortx,driver>=525,driver<526 brand=titan,driver>=525,driver<526 brand=titanrtx,driver>=525,driver<526"
-ENV NV_CUDA_CUDART_VERSION 12.1.105-1
+ENV NVARCH sbsa
```

Comment:
```suggestion
ENV NVARCH x86_64
```

Reply:
tnx for noticing that, i had this change on my repo but i never push it :facepalm: Opened a new PR here: https://github.com/opendatahub-io/notebooks/pull/717